modules/transactions/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     implementation project(':ignite-workers')
     implementation project(':ignite-low-watermark')
     implementation project(':ignite-system-view-api')
+    implementation project(':ignite-failure-handler')
```

Comment:
Why isn't it in the api section?

Reply:
Which one is in the public API of this module? `FailureProcessor`? If yes, then disregard my comment.

Reply:
`FailureProcessor` comes from `ignite-core`. How is it related to `ignite-transactions` API?

Reply:
If `FailureProcessor` is part of a public constructor, then its a part of this module's API